jina/hubble/helper.py

```diff
@@ -484,10 +490,50 @@ def is_requirements_installed(
         return isinstance(ex, VersionConflict)
     return True
 
+def get_requirements_env_variables(requirements_file: 'Path') -> list:
+    """get the env variables in requirements.txt
+    :param requirements_file: the requirements.txt file
+    :return: List of env variables in requirements.txt
+    """
+    env_variables = []
+    with requirements_file.open() as requirements:
+        for req in requirements:
+            req = req.strip()
+            if (not req) or req.startswith('#'):
+                continue
+            else:
+                variables = get_env_variables(req)
+                env_variables.extend(variables)
 
-def _get_install_options(requirements_file: 'Path', excludes: Tuple[str] = ('jina',)):
-    from .requirements import parse_requirement
+    return env_variables
 
+def check_requirements_env_variable(env_variable: str) -> bool:
+    """ 
+    check the env variable is limited
+    to uppercase letter and number and the `_` (underscore).
+    :param env_variable: env_variable in the requirements.txt file
+    :return: True or False if not satisfied
+    """
+    return check_env_variable(env_variable)
+
+def replace_env_variables(requirements_file: 'Path') -> list:
+    """get the env variables in requirements.txt
```

Comment:
This comment is copied from `get_requirements_env_variables`. Should be changed accordingly.